File: src/templates/java-compat-fc/src/main/java/com/TempName/TempNameViewManagerImpl.java
package com.TempName;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import com.facebook.react.uimanager.ThemedReactContext;


class TempNameViewManagerImpl {
    
    public static final String NAME = "TempName";

    public static TempName createViewInstance(@NonNull ThemedReactContext context) {
        return new TempName(context);
    }

    public static void setText(TempName view, @Nullable String text) {
        view.setText(text);
    }
}

File: src/templates/kotlin-compat-fc/src/main/java/com/TempName/TempNameViewManagerImpl.kt
package com.TempName

import com.facebook.react.uimanager.ThemedReactContext

import com.TempName.*

object TempNameViewManagerImpl {
  const val NAME = "TempName";

  fun createViewInstance(context: ThemedReactContext?) = TempName(context);

  fun setText(view: TempName?, text: String?) {
    view?.setText(text)
  }
}